Tutorial/GettingStartedGuide.md

```diff
@@ -0,0 +1,148 @@
+# Getting Started Guide  
+  
+This guide explains how to package up your trained model as a web service with a predict API that can be scanned using Certifai
+  
+  
+## Table of Contents  
+1. [pre-requisites](#pre-requisites)   
+2. [predict](#predict)  
+3. [api](#api)  
+4. [build-container](#build-container)  
+5. [run-container](#run-container)  
+5. [test-container](#test-container)  
+  
+## pre-requisites  
+  
+1. trained model
```

Comment:
If someone is stepping through this tutorial, how do they get a trained model?

Reply:
there is no init function in the base s2i builder code, it just adds routes and starts the daemon.

for getting a trained model, i explain in the docstring of function that loads the model, that user may decide to bring in external binaries or just use plain sklearn trained objects. Will add that to the doc as well